fwlitetesteval.sh

```diff
@@ -0,0 +1,2 @@
+ls -d all_* | while read -r d; do nd=`ls $d | grep -c png`; echo -e $nd" \t "$d; done 
+# ls -d all_* | while read -r d; do nd=`ls $d | grep -c png`; echo -e $nd" \t "$d; done | grep  -v ^0
```

Comment:
it's worth passing the -v ^0 part to be an argument.
if the intent is to do the grep out outside, then just remove this line